Packs/ThreatIntelligenceManagement/Dashboards/dashboard-Threat_Intelligence_Management.json

```diff
@@ -1,665 +1,665 @@
-							"key": "expirationStatus",
-							"position": 14,
-							"width": 175
-						},
-						{
-							"isDefault": true,
-							"key": "expiration",
-							"position": 15,
-							"width": 190
-						}
-					]
-				},
-				"size": 0,
-				"category": ""
-			}
-		},
-		{
-			"id": "4614e260-36d1-11ea-bac2-cb1342f8efbb",
-			"forceRange": false,
-			"x": 0,
-			"y": 1,
-			"i": "4614e260-36d1-11ea-bac2-cb1342f8efbb",
-			"w": 3,
-			"h": 1,
-			"widget": {
-				"id": "530fa858-2e59-4ef9-8b44-f47db1011b36",
-				"version": 1,
-				"modified": "2020-01-14T13:24:57.840603414Z",
-				"name": "Number of Expired Indicators",
-				"dataType": "indicators",
-				"widgetType": "trend",
-				"query": "expirationStatus:expired",
-				"sort": null,
-				"isPredefined": false,
-				"dateRange": {
-					"fromDate": "0001-01-01T00:00:00Z",
-					"toDate": "0001-01-01T00:00:00Z",
-					"period": {
-						"by": "",
-						"byTo": "",
-						"byFrom": "days",
-						"toValue": null,
-						"fromValue": 7,
-						"field": ""
-					},
-					"fromDateLicense": "0001-01-01T00:00:00Z"
-				},
-				"params": {
-					"tableColumns": [
-						{
-							"isDefault": true,
-							"key": "indicator_type",
-							"position": 1,
-							"width": 120
-						},
-						{
-							"isDefault": true,
-							"key": "value",
-							"position": 2,
-							"width": 300
-						},
-						{
-							"isDefault": true,
-							"key": "score",
-							"position": 4,
-							"width": 120
-						},
-						{
-							"isDefault": true,
-							"key": "firstSeen",
-							"position": 5,
-							"width": 275
-						},
-						{
-							"isDefault": true,
-							"key": "lastSeen",
-							"position": 6,
-							"width": 275
-						},
-						{
-							"isDefault": true,
-							"key": "modifiedTime",
-							"position": 7,
-							"width": 180
-						},
-						{
-							"isDefault": true,
-							"key": "timestamp",
-							"position": 8,
-							"width": 190
-						},
-						{
-							"isDefault": true,
-							"key": "investigationIDs",
-							"position": 9,
-							"width": 150
-						},
-						{
-							"isDefault": true,
-							"key": "comment",
-							"position": 10,
-							"width": 175
-						},
-						{
-							"isDefault": true,
-							"key": "expirationStatus",
-							"position": 14,
-							"width": 175
-						},
-						{
-							"isDefault": true,
-							"key": "expiration",
-							"position": 15,
-							"width": 190
-						}
-					]
-				},
-				"size": 0,
-				"category": ""
-			}
-		},
-		{
-			"id": "0295b860-36d2-11ea-bac2-cb1342f8efbb",
-			"forceRange": false,
-			"x": 3,
-			"y": 2,
-			"i": "0295b860-36d2-11ea-bac2-cb1342f8efbb",
-			"w": 3,
-			"h": 2,
-			"widget": {
-				"id": "e0aded97-34a1-45f3-8b5b-a54b43c13f5f",
-				"version": 1,
-				"modified": "2020-01-14T13:30:13.387314353Z",
-				"name": "Active Indicators by Reputation",
-				"dataType": "indicators",
-				"widgetType": "pie",
-				"query": "(expirationStatus:active or expirationStatus:\"\")",
```

Comment:
This is the only change, adding parenthesis, not sure why it thinks it all changed